io/plugins/eu.esdihumboldt.hale.io.xls.test/src/eu/esdihumboldt/hale/io/xls/test/XLSInstanceIOTest.java

```diff
@@ -92,7 +91,7 @@ public void test() {
 		// read the instances form the temporary XLS file
 		XLSInstanceReader reader = new XLSInstanceReader();
 		reader.setSourceSchema(schema);
-		reader.setParameter(CommonSchemaConstants.PARAM_SKIP_FIRST_LINE, Value.of(true));
+		reader.setParameter(CommonSchemaConstants.PARAM_SKIP_N_LINES, Value.of(1));
```

Comment:
It would be great to not only have a test for the new functionality but also for the old one to test backwards compatibility.

Reply:
I hope that here the functionality with `boolean` parameter `True`was tested as I don't see any test case testing the backwards compatibility with a boolean parameter.

Reply:
Added.